### PR TITLE
OHRM-542: Update Dockerfile to support OrangeHRM 4.3.4

### PR DESCRIPTION
**Pull request by RajithaKumara**
Get the patch:
- run `$ wget https://patch-diff.githubusercontent.com/raw/orangehrm/orangehrm/pull/624.patch`
- apply the patch to the master branch (`$ git apply 624.patch`)

Steps to run:
- run `$ docker build -t orangehrm/orangehrm:4.3.5 .`
- run `$ docker run -d --name ohrm-435 orangehrm/orangehrm:4.3.5`
- run `$ docker inspect ohrm-435 | grep IPAddress` and copy `IPAddress`
- Access OrangeHRM through browser using above `IPAddress` with 80 port
- If got installer UI, run DB container
- run `docker pull mariadb:10.4`
- run `$ docker run -d --name mariadb-104 -e MYSQL_ROOT_PASSWORD=<pass> mariadb:10.4`
- run `$ docker inspect mariadb-104 | grep IPAddress` and copy `IPAddress`
- Check `ohrm-434` container IP and `mariadb-104` container IP in same subnet.
- Go ahead install OrangeHRM though browser.

This fix applies for #601 and #407

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,62 +1,66 @@
...
+	apt-get install -y --no-install-recommends \
+		libfreetype6-dev \
+		libjpeg-dev \
+		libpng-dev \
+		libzip-dev \
+		unzip \
+	; \
+	\
+	cd .. && rm -r html; \
+	curl -fSL -o orangehrm.zip "https://sourceforge.net/projects/orangehrm/files/stable/${OHRM_VERSION}/orangehrm-${OHRM_VERSION}.zip/download"; \
```
Comment (samanthajayasinghe) [inline Dockerfile]: any possible to get it via git release?

File: Dockerfile [hunk 2]
```diff
@@ -1,62 +1,66 @@
...
+	chmod -R 775 html/symfony/cache html/symfony/log; \
+	\
+	docker-php-ext-configure gd \
+		--with-freetype-dir=/usr \
+		--with-png-dir=/usr \
+		--with-jpeg-dir=/usr \
+	; \
+	\
+	docker-php-ext-install -j "$(nproc)" \
+		gd \
```
Comment (samanthajayasinghe) [inline Dockerfile]: What about ioncube?

File: Dockerfile (hunk 1 above)
Comment (RajithaKumara) [reply Dockerfile]: It's possible. But after release each version.

File: Dockerfile (hunk 2 above)
Comment (RajithaKumara) [reply Dockerfile]: ionCube and LDAP are in todo list

File: Dockerfile (hunk 1 above)
Comment (RajithaKumara) [reply Dockerfile]: Done

File: Dockerfile (hunk 2 above)
Comment (RajithaKumara) [reply Dockerfile]: Done

File: Dockerfile [hunk 3]
```diff
@@ -1,62 +1,80 @@
...
-# Copy Supervisor configuration
-ADD docker-build-files/supervisord.conf /etc/supervisor/conf.d/supervisord.conf
-
-# Start apache/mysql
-CMD /usr/bin/supervisord -c /etc/supervisor/conf.d/supervisord.conf
-
-
+FROM php:7.3.6-apache
+
+ENV OHRM_VERSION 4.3.4
```
Comment (samanthajayasinghe) [inline Dockerfile]: @RajithaKumara can you change this to support 4.3.5?

Comment (RajithaKumara) [reply Dockerfile]: Sure. Then we supporting new Dockerfile from 4.3.5?

Comment (RajithaKumara) [reply Dockerfile]: Done

**Comment by samanthajayasinghe** [comment]
@RajithaKumara I still your commit and push to master .. sorry :(

**Pull request closed without merging**